tasks/sast-go.yaml

```diff
@@ -0,0 +1,39 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: gosec-check
+spec:
+  workspaces:
+    - name: workspace
+  results:
+    - description: Test output
+      name: HACBS_TEST_OUTPUT
+  steps:
+    - name: gosec-check
+      image: quay.io/redhat-appstudio/hacbs-test:stable
+      workingDir: $(workspaces.workspace.path)/hacbs/$(context.task.name)
+      script: |
+        /usr/local/go/bin/gosec -no-fail -fmt=sarif -out=gosec_output.json $(workspaces.workspace.path)/... | tee gosec_output.txt
+        GOSEC_EXIT_CODE=$?
```

Comment:
by default the script will use `bash -ex` which means that if $? is different then 0 then it will fail.

Reply:
Will it? I couldn't simulate that, is that not the default on either CRC or the quicklab clusters?

Reply:
Doesn't piping to `tee` overrule the exit code, so it should always be zero anyways?

Reply:
@dirgim you're right: 
```bash
$ false | tee
$ echo $?
0
$ false
$ echo $?
1
```